src/cloud-functions.ts

```diff
@@ -53,38 +55,92 @@ export interface Event {
+   * * `USER` Known user.
+   * * `UNAUTHENTICATED` Unauthenticated action
+   * * `null` For event types that do not provide user information (all except
+   *   Realtime Database).
+   *
    */
   authType?: 'ADMIN' | 'USER' | 'UNAUTHENTICATED';
+
   /**
-   * ID of the event
+   * The event’s unique identifier.
    */
   eventId: string;
+
   /**
-   * Type of event
+   * Type of event. Valid values are:
+   *
+   * * `providers/google.firebase.analytics/eventTypes/event.log`
+   * * `providers/firebase.auth/eventTypes/user.create`
+   * * `providers/firebase.auth/eventTypes/user.delete`
+   * * `providers/firebase.crashlytics/eventTypes/issue.new`
+   * * `providers/firebase.crashlytics/eventTypes/issue.regressed`
+   * * `providers/firebase.crashlytics/eventTypes/issue.velocityAlert`
+   * * `providers/google.firebase.database/eventTypes/ref.write`
+   * * `providers/google.firebase.database/eventTypes/ref.create`
+   * * `providers/google.firebase.database/eventTypes/ref.update`
+   * * `providers/google.firebase.database/eventTypes/ref.delete`
+   * * `providers/cloud.firestore/eventTypes/document.write`
+   * * `providers/cloud.firestore/eventTypes/document.create`
+   * * `providers/cloud.firestore/eventTypes/document.update`
+   * * `providers/cloud.firestore/eventTypes/document.delete`
+   * * `google.pubsub.topic.publish`
+   * * `google.storage.object.finalize`
+   * * `google.storage.object.archive`
+   * * `google.storage.object.delete`
+   * * `google.storage.object.metadataUpdate`
+   * * `google.firebase.remoteconfig.update`
    */
   eventType: string;
+
   /**
-   * Key-value pairs that represent the values of wildcards in a database
-   * reference. Cannot be accessed while inside the handler namespace.
+   * An object containing the values of the wildcards in the `path` parameter
+   * provided to the [`ref()`](functions.database#.ref) method for a Realtime
+   * Database trigger. Cannot be accessed while inside the handler namespace.
    */
   params: { [option: string]: any };
+
   /**
-   * Resource that triggered the event
+   * The resource that emitted the event. Valid values are:
+   *
+   * * Analytics &mdash; `projects/<projectId>/events/<analyticsEventType>`
+   * * Realtime Database &mdash;
+       `projects/_/instances/<databaseInstance>/refs/<databasePath>`
+  * * Storage &mdash;
+      `projects/_/buckets/<bucketName>/objects/<fileName>#<generation>`
+  * * Authentication &mdash; `projects/<projectId>`
+  * * Pub/Sub &mdash; `projects/<projectId>/topics/<topicName>`
+  *
+  * Because Realtime Database instances and Cloud Storage buckets are globally
+  * unique and not tied to the project, their resources start with `projects/_`.
+  * Underscore is not a valid project name.
+  *
    */
   resource: Resource;
   /**
-   * Timestamp for when the event ocurred (ISO 8601 string)
+   * Timestamp for the event as an
+   * [RFC 3339](https://www.ietf.org/rfc/rfc3339.txt) string.
+   *
```

Comment:
Not sure about whether this one changes how the docs look but if it doesn't, let's remove this extra line.